includes/ziti/ziti.h

```diff
@@ -518,6 +518,26 @@ extern int ziti_accept(ziti_connection clt, ziti_conn_cb cb, ziti_data_cb data_c
 ZITI_FUNC
 extern int ziti_close(ziti_connection *conn);
 
+/**
+ * @brief Closes the outgoing (write) side of the given ziti connection.
+ *
+ * Any pending write requests will be able to complete. The `conn` should refer to a initialized ziti connection.
+ *
+ * This notifies peer ziti connection that no more data will be sent -- peer receives EOF.
+ * Any further calls to `ziti_write()` will return an error.
+ *
+ * This leaves incoming(read) side of ziti connection open.
+ *
+ * @note this is roughly equivalent to calling `uv_shutdown()` on a duplex `uv_stream`, or
+ *       `shutdown(sock, SHUT_WR)` on socket fd.
+ *
+ * @param conn the #ziti_connection to be closed
+ *
+ * @return #ZITI_OK or corresponding #ZITI_ERRORS
+ */
+ZITI_FUNC
+extern int ziti_close_write(ziti_connection conn);
```

Comment:
i don't know about this, it seems odd to me. if i'm the one using this sdk why would i want to have `ziti_close_write`. Why wouldn't I just "not write to the connection" any more? is this function really needed?

Reply:
the reason is to notify the other side that you're done

Reply:
see matching PR for Go SDK https://github.com/openziti/sdk-golang/pull/99